:recycle: restructure addition docs (#145)
neo4j/docs-ops-manager: modules/ROOT/pages/addition/agent-installation/agent-install.adoc

= Install agent

* Download platform specific binary: https://neo4j.com/download-center/#ops-manager[Download]
+

> NOTE: If downloading agents is not a option in some environments, NOM agent binaries are also bundled with Neo4j packages.
> NOM agent packages can be found in the `products` folder in the Neo4j home directory. 
> These binaries have support for the Neo4j version that they are bundled with.

Reviewer: ```suggestion
[NOTE]
==== 
In certain environments, downloading agents is not an option.
Therefore, NOM agent binaries are also bundled with Neo4j packages.
NOM agent packages can be found in the `products` folder in the Neo4j home directory. 
These binaries have support for the Neo4j version that they are bundled with.
====
```
